src/test/resources/app.properties

```diff
@@ -1,5 +1,8 @@
 email = qaworkshopsv@mailinator.com
 password = password
+emailValid = qaworkshopsv@mailinator.com
+passwordValid = QAWorkshopSV1!
+product1 = /grey-long-faux-fur-coat/
```

Comment:
So we don't load this property files with all the tests input data, create a new properties files for product related tests. Going forward we'll have multiple properties files for different tests categories (e.g. tests related to product will have their prop files, tests related to an order will have their own prop files, etc.)
Also give the 'product1' property name a more suggestive name.

Reply:
done